src/app/search/search.component.ts

```diff
@@ -67,6 +84,9 @@ export class SearchComponent implements OnInit {
 
   private getCaseFilterFromFormGroup(formGroup?: FormGroup): object {
     const result = {};
+    result[SearchComponent.METADATA_FILTER] = {};
```

Comment:
feels like things are getting complicated here. 
To simplify a bit the code, can we use types maybe? i.e. introducing  a class which has  2 properties caseFilter and metadataFilter and returning that instead of an Object

Reply:
this code fragment is from another ticket and got merged here accidentally